exporters/stdout/stdoutmetric/example_test.go

```diff
@@ -257,8 +257,8 @@ func Example() {
 	//                   6,
 	//                   0
 	//                 ],
-	//                 "Min": {},
-	//                 "Max": {},
+	//                 "Min": null,
+	//                 "Max": null,
```

Comment:
nit: I think we can update the `mockData` to have meaningful example.

Reply:
I've added separate entry called `requests.size` as I think we should have one example without specific values to check for proper `nil` marshaling.